### ISSUE TITLE
Greenbone outdated

### ISSUE THREAD

**Issue by GORGES** [yu210148/gvm_install#93]
My Greenbone scanner is version 21, and there are now notices that we should be on version 22, which was released July 2022. Any chance this "gvm_install" script will be updated for version 22?

**Comment by yu210148** [comment]
Hi Gorges,

I don't have time to look at this very often any more but would happily accept any pull requests that kept it up to date.

Kev.

**Comment by GORGES** [comment]
Fair enough! I'll see if I can find some time myself to see if I can update the script.

**Comment by yu210148** [comment]
Trust would be great! When they went from version 20 to 21 they changed how some of the components were compiled I took a stab at updating it myself but it was one of the contributors that actually got it working. Hopefully they didn't do that again. :)

**Comment by itiligent** [comment]
For those looking, I've built a new build script that is up to date and current with Greenbones install instructions  (also works  for Debian 12) A lot of this script is legacy now. The script also adds an nginx front end with self signed SSL to keep things simple rather than deal with crytic Greenbone SSL. Theres a source build or docker build script (greenbones's official containers) here:
[https://github.com/itiligent/GVM-Install](url)
